cfn-resources/project/cmd/resource/resource.go

```diff
@@ -181,7 +184,7 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (event h
 		return event, nil
 	}
 
-	if currentModel.Name != nil {
+	if currentModel.Name != nil || !reflect.DeepEqual(prevModel.Tags, currentModel.Tags) {
```

Comment:
Given that maps do not have a fixed order, could we have scenarios where having the same tags still results in reflect.DeepEqual returning false? Don't see a big concern but maybe we can make a more precise check if it is the case.

Reply:
I think updating the project "too many times" is not a concern. Also, I think DeepEqual is accurate.
Maybe @lantoli knows the details better.

WDYT?

Reply:
yes I think DeepEqual should work fine.   (on the other hand DeepEqual is not normally recommended to use in Prod , only in tests, as it uses reflection and performance is not very good, but shouldn't be any issue in our case)

Reply:
Marking as resolved. Feel free to re-open